src/zope/app/authentication/browser/rolepermissionview.py

```diff
@@ -116,7 +118,8 @@ def update(self, testing=None):
                         elif setting == Deny.getName():
                             prm.denyPermissionToRole(rperm, rrole)
                         else:
-                            raise ValueError("Incorrect setting: %s" % setting) # pragma: no cover
+                            raise ValueError("Incorrect setting: %s" %
+                                             setting)  # pragma: no cover
```

Comment:
I would move the `%` to the continuation line as well.  IIRC flake8 doesn't complain by default because PEP-8 switched from recommending operators on the previous line to recommending operators on the next line at some point within the last couple of years.

Reply:
Okay.